src/finbars/finbars.ts

```diff
@@ -863,10 +863,10 @@ const defaultNormals: WheelNormals = {
         moz: 35
     },
     win: {
-        webkit: 2.6,
+        webkit: 3,
```

Comment:
The original value no longer gives you a consistent scrolling experience. It does move approximately 3 rows down and up, but it's not an exact one so what I observed is you get a "net" scroll up. This means the more you repeat scrolling up and down the higher the scroll will be in the end.

I suspect it's to do with the fact we no longer have the bottom scroll gutter